lib/doc/worksheet.js

```diff
@@ -563,6 +563,23 @@ class Worksheet {
   }
 
   // =========================================================================
+
+  // Style fill
+  fillCells(range, fill, makeClones = true) {
```

Comment:
I find this function signature confusing to read, like in the README:
```js
ws.fillCells('A1:C5', fill, false)
```
I'd suggest to use `fillCells(range, fill, {makeClones: true})` instead.

Reply:
good feedback, thanks